build.sbt

```diff
@@ -327,6 +327,16 @@ lazy val zincScripted = (project in internalPath / "zinc-scripted").
   ).
   configure(addSbtUtilScripted)
 
+lazy val crossTestBridges = {
+  Command.args("crossTestBridges", "<version>") { (state, args) =>
+    require(args.nonEmpty, "Missing Scala version argument.")
+    compilerBridgeScalaVersions.flatMap { (bridgeVersion: String) =>
+      s"wow $bridgeVersion" ::
+        s"${compilerBridge.id}/test" :: Nil
```

Comment:
The reason why we don't is because `plz` causes `OutOfMemoryError`. I've actually experienced this myself locally and there's a comment in `publishBridgesAndTest` that makes reference to it.